packages/terra-date-picker/src/terra-dev-site/doc/example/CalendarFilter/CalendarFilterFilterDates.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import moment from 'moment';
+import CalendarFilter from 'terra-date-picker/lib/CalendarFilter';
+
+const CalendarFilterFilterDates = () => {
+  const isWeekday = (dateValue) => {
+    const momentDate = moment(dateValue);
+
+    if (momentDate && momentDate.isValid()) {
+      const day = momentDate.day();
+      return day !== 0 && day !== 6;
```

Comment:
Since this is an example used by consumers, would it be helpful to add an inline doc to describe not Sunday or Saturday? I had to look up the moment docs to understand this fully: https://momentjs.com/docs/#/get-set/day/

Reply:
updated : [34148ea0fc](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-0411a587b98904714fd5ca34148ea0fc)